src/components/common/toast/toasts.tsx

```diff
@@ -1,14 +1,19 @@
+import { ToastWrapper } from "src/form-builder.styles";
 import { useDisplay } from "../../../context-providers";
 import { DisplayToast } from "./toast";
 
-export const Toasts = () => {
+export const Toasts: Function = () => {
```

Comment:
shouldnt be returning `JSX.Element`? if it should, there is no need to declare the return type too since it will be inferred

Reply:
has been updated in the CCUBE-1353 PR